apstools/callbacks/callback_base.py

```diff
@@ -10,8 +10,9 @@
 
 import datetime
 import logging
-import pathlib
+from typing import Any, Dict, List, Optional, Union
```

Comment:
Is there an advantage to use `typing.Dict[str, str]` when `dict[str, str]` passes linting and testing?  Same for `typing.List`.

Reply:
I don't think there is any effect difference here. Here is why I started using typing as default:
https://stackoverflow.com/questions/37087457/difference-between-defining-typing-dict-and-dict

Reply:
Forget my previous post. After 3.9 or 3.10 this are the same. I will move back to standard types.